main.js

```diff
@@ -33,6 +34,8 @@ async function main() {
 
     await install(version);
 
+    core.info(`::add-matcher::${path.join(__dirname, "problem-matchers.json")}`);
```

Comment:
The [example on the `action/toolkit` repo only uses a relative path](https://github.com/actions/toolkit/blob/main/docs/commands.md#problem-matchers). Isn't it possible to just use a relative path from the workspace root here?

```suggestion
    core.info("::add-matcher::problem-matchers.json");
```

If we need to use a full path, the code should likely be updated to ESM now that this file is now `main.mjs`:

```ts
import path from "node:path";
import process from "node:process";
...
```

```suggestion
    core.info(`::add-matcher::${path.join(import.meta.dirname, "problem-matchers.json")}`);
```

Reply:
I tried to change this to relative path, but this did not work. I see that the path is calculated relative to the root of the repository where this action is used.

```
Run r7kamura/setup-deno@problem-matchers
Going to install stable version 1.46.3.
Downloading Deno from https://github.com/denoland/deno/releases/download/v1.46.3/deno-x86_64-unknown-linux-gnu.zip.
/usr/bin/unzip -o -q /home/runner/work/_temp/f448c3f2-3868-44c3-a8c0-d0d3c95c533d
Cached Deno to /opt/hostedtoolcache/deno/1.46.3/x64.
Error: Unable to process command '::add-matcher::problem-matchers.json' successfully.
Error: Could not find file '/home/runner/work/bump-request/bump-request/problem-matchers.json'.
Installation complete.
```

- https://github.com/r7kamura/bump-request/actions/runs/12051024336/job/33601146503?pr=12

Reply:
Ahh, alright, so targeted more towards people writing workflows rather than actions. Cool that you checked!